datafusion/wasmtest/README.md

```diff
@@ -32,7 +32,7 @@ Some of DataFusion's downstream projects compile to WASM to run in the browser.
 
 ## Setup
 
-First, [install wasm-pack](https://rustwasm.github.io/wasm-pack/installer/)
```

Comment:
TIL https://blog.rust-lang.org/inside-rust/2025/07/21/sunsetting-the-rustwasm-github-org/